OpenEdXMobile/res/values/strings.xml

```diff
@@ -36,9 +36,6 @@
     <string name="cannot_open_url">Unable to open web page.</string>
 
     <!-- Course Sharing -->
-
-    <!-- Description of button that will open the share course menu -->
-    <string name="share_course_button">Share your course</string>
```

Comment:
we can use this string as an icon description for accessibility. better to revert this change. 

same with the translations.

Reply:
That was my intention as well by `ImageSpan.setContentDescription()` is for API level 31+. That's why I wasn't able to utilize this string.